Code/Tools/AssetBundler/Platform/Linux/source/utils/GUIApplicationManager_Linux.cpp

```diff
@@ -12,6 +12,6 @@ namespace Platform
 {
     AzQtComponents::WindowDecorationWrapper::Option GetWindowDecorationWrapperOption()
     {
-        return AzQtComponents::WindowDecorationWrapper::OptionDisabled;
+        return AzQtComponents::WindowDecorationWrapper::OptionNone;
```

Comment:
@spham-amzn  was there a reason we needed to disable this on Linux?

Reply:
Not that I am aware of

Reply:
Cool thanks for weighing in!